app/controllers/service_types_controller.rb

```diff
@@ -3,8 +3,40 @@ class ServiceTypesController < ApplicationController
 
   # Authentication by default inherited from ApplicationController.
 
+  before_action :authorize_for_districtwide_access_admin, except: [:index]
```

Comment:
@alexsoble This line was surprising to me at first, until I realized these endpoints are unrelated bits of code.  What do you think about cutting-and-pasting this feature into its own controller so it's safely isolated, and can slap its authorization check on everything?